client/client-lib/src/api.rs

```diff
@@ -457,8 +457,8 @@ where
         .await
     }
 
-    async fn fetch_last_epoch(&self) -> FederationResult<u64> {
-        self.request_eventually_consistent("/epoch".to_owned(), erased_no_param())
+    async fn fetch_epoch_count(&self) -> FederationResult<u64> {
+        self.request_eventually_consistent("/fetch_epoch_count".to_owned(), erased_no_param())
```

Comment:
nit: we might want some unified naming scheme. `get` seems more common. thoughts @mxz42? Not a blocker though.

Reply:
yeah, as dpc [mentioned](https://github.com/fedimint/fedimint/pull/1566#discussion_r1094912734) `getepochcount` would be best to be sort of close to the bitcoin cli